### PR TITLE
Add detailed docs

### PR DESCRIPTION
**Pull request by kadora626**
## Purpose
_Describe the problem or feature in addition to a link to the issues._

Add more detailed qramana documentation to docs.

## Approach
_How does this change address the problem?_

Add index.md

#### Open Questions and Pre-Merge TODOs
- [ ] Use github checklists. When solved, check the box and explain the answer.

## Learning
_Describe the research stage_

_Links to blog posts, patterns, libraries or addons used to solve this problem_

#### Blog Posts
- [How to Pull Request](https://github.com/flexyford/pull-request) Github Repo with Learning focused Pull Request Template.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# qramana: A typescript library for quantum calculation.
```
Comment (imatsuzaki) [inline docs/index.md]: 細かいけど `TypeScript` で統一しちゃったほうがいいかな。

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+`qramana` is a typescript library for quantum calculation.
```
Comment (imatsuzaki) [inline docs/index.md]:
```suggestion
`qramana` is a TypeScript library for quantum calculation.
```

File: docs/index.md (hunk 1 above)
Comment (kadora626) [reply docs/index.md]: 何も考えないで打ち込んでた。統一します

**Comment by kamakiri01** [comment]
細かい加筆とREADMEに一部を転記しました。

**Pull request merged**